index.html

```diff
@@ -352,46 +353,49 @@ <h3 id="abtMeText" class="abtMeText">I can help you build a product,<br>feature
       </div>
       <p class="abtMeCardTitle">Skills</p>
       <ul class="abtMeCardList2">
-        <li>
+        <li class="skillsTag">
           <button type="button" class="langButton">
             Codekit
           </button>
         </li>
-        <li>
+        <li class="skillsTag">
           <button type="button" class="langButton">
             GitHub
           </button>
         </li>
-        <li>
+        <li class="skillsTag">
           <button type="button" class="langButton">
             Codepen
           </button>
         </li>
-        <li>
+        <li class="skillsTag">
           <button type="button" class="langButton">
             Gitlab
           </button>
         </li>
-        <li>
-          <button type="button" class="langButton secondOrderBttn2" id="terminalTag">
+        <li class="skillsTag">
+          <button type="button" class="langButton" id="terminalTag">
             Terminal
           </button>
         </li>
       </ul>
     </div>
+  </section>
+  <section id="contact">
     <div id="contactMeTextPH">
-      <p id="contactMeText">
+      <h2 id="contactMeText">
         I'm always interested
         in hearing about new projects,
         so if you'd like to chat please
         get in touch.
-      </p>
+      </h2>
     </div>
     <div id="contactMeBlankSpace"></div>
-    <form id="contactForm">
-      <input type="text" name="FullName" id="nameFormBox" class="formBox1" placeholder="Full Name">
-      <input type="text" name="EmailAddress" id="mailFormBox" class="formBox1" placeholder="Email Adress">
-      <input type="text" name="EmailAddress" id="commentsFormBox" class="formBox2" placeholder="Enter Text Here">
+    <form id="contactForm" action="https://formspree.io/f/moqbvdgn" method="post">
+      <input type="text" name="name" id="nameFormBox" class="formBox1" placeholder="Full Name" maxlength="30" required>
+      <input type="email" name="email" id="mailFormBox" class="formBox1" placeholder="Email Adress" required>
+      <textarea name="comments" id="commentsFormBox" class="formBox2"
+        maxlength="500" required>Any comments?</textarea>
       <button type="submit" id="submitFormBttn">Get in touch</button>

```

Comment:
- [ ] Please decrease the width of the submit button and make sure it matches the Figma design.

### Your design
![image](https://user-images.githubusercontent.com/83803414/187219064-45c03fb2-5420-4c3e-820b-88f4b6f136ce.png)

### Figma design
![image](https://user-images.githubusercontent.com/83803414/187219124-15b7b422-651c-4be3-92b4-9af147b4ce2a.png)